src/main/java/net/hypixel/nerdbot/command/UserCommands.java

```diff
@@ -198,4 +217,253 @@ public static Date parseCustomFormat(String time) throws DateTimeParseException
         Instant date = Instant.now().plus(duration);
         return Date.from(date);
     }
+
+    @JDASlashCommand(name = "suggestions", subcommand = "by-member", description = "View user suggestions.")
+    public void mySuggestions(
+        GuildSlashEvent event,
+        @AppOption @Optional Integer page,
+        @AppOption(description = "Member to view.") @Optional Member member,
+        @AppOption(description = "Tags to filter for (comma separated).") @Optional String tags,
+        @AppOption(description = "Words to filter title for.") @Optional String title,
+        @AppOption(description = "Toggle alpha suggestions.") @Optional Boolean alpha
+    ) {
+        BotConfig config = NerdBotApp.getBot().getConfig();
+        page = (page == null) ? 1 : page;
+        final int pageNum = Math.max(page, 1);
+        final Member searchMember = (member == null) ? event.getMember() : member;
+        final boolean isAlpha = (alpha != null && alpha);
+        final String[] suggestionForumIds = (alpha != null && alpha) ? config.getAlphaSuggestionForumIds() : config.getSuggestionForumIds();
+
+        if (suggestionForumIds == null || suggestionForumIds.length == 0) {
+            event.reply("No " + (isAlpha ? "alpha " : "") + "suggestion forums are setup in the config.").setEphemeral(true).queue();
+            return;
+        }
+
+        List<Suggestion> suggestions = getSuggestions(suggestionForumIds, searchMember, tags, title, isAlpha);
+
+        if (suggestions.isEmpty()) {
+            event.reply("You have no suggestions matched with tags " + (tags != null ? ("`" + tags + "`") : "*ANY*") + " or title containing " + (title != null ? ("`" + title + "`") : "*ANYTHING*") + ".").setEphemeral(true).queue();
```

Comment:
Can you rephrase this message 

Maybe something like this to keep it simple:
Found no suggestions matching the specified filters!

Alternatively just make the formatting of the message better when there's no tags/title etc